packages/ragbits-evaluate/src/ragbits/evaluate/pipelines/__init__.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+from omegaconf import DictConfig
+
+from ragbits.core.utils.config_handling import import_by_path
+
+from .base import EvaluationPipeline
+
+module = sys.modules[__name__]
+
+
+def pipeline_factory(pipeline_config: DictConfig) -> EvaluationPipeline:
```

Comment:
maybe we can leverage new `WithConstructionConfig` class here, same applies for merics and dataloaders factories